_gabarits-jsonld/README.md

```diff
@@ -58,18 +70,24 @@ Indiquez si un événement est [programmé](https://schema.org/EventScheduled) (
 Indiquez si un événement se produit [en personne](https://schema.org/OfflineEventAttendanceMode), [en ligne](https://schema.org/OnlineEventAttendanceMode), ou format [hybride](https://schema.org/MixedEventAttendanceMode).
 
 ### [_organizer_](https://schema.org/organizer)
-Saisissez les informations identifiant et décrivant l'organisme qui est responsable de la présentation du spectacle. Il peut s'agir d'un organisme diffuseur ou d'une compagnie qui présente sa propre production. On peut ajouter plusieurs _organizer(s)_. Les entités décrites sous cette propriété sont généralement de @type [Organization](https://schema.org/Organization).
+Saisissez les informations identifiant et décrivant l'organisme qui est responsable de la présentation du spectacle. Il peut s'agir d'un organisme diffuseur ou d'une compagnie qui présente sa propre production. On peut ajouter plusieurs _organizer(s)_. Les _entitées imbriquées_[^1] sous cette propriété sont généralement de @type [Organization](https://schema.org/Organization).
 
 ### [_performer_](https://schema.org/performer)
-Saisissez les informations identifiant et décrivant la compagnie, le groupe ou la (des) personne(s) qui est(sont) responsable de l'exécution de la représentation. Il est possible d'ajouter plusieurs _performer(s)_. Les entités décrites sous cette propriété peuvent être de @type [Organization](https://schema.org/Organization), [PerformingGroup](https://schema.org/PerformingGroup) ou [Person](https://schema.org/Person). 
+Saisissez les informations identifiant et décrivant la compagnie, le groupe ou la (des) personne(s) qui est(sont) responsable de l'exécution de la représentation. Il est possible d'ajouter plusieurs _performer(s)_. Les entités imbriquées sous cette propriété peuvent être de @type [Organization](https://schema.org/Organization), [PerformingGroup](https://schema.org/PerformingGroup) ou [Person](https://schema.org/Person). 
 
 ### [_location_](https://schema.org/location)
-Saisissez les informations identifiant et décrivant l'endroit où est présenté l'événement. L'entité décrite sous la propriété _location_ peut être de @type [Place](https://schema.org/Place) (un lieu physique) ou de @type [VirtualLocation](https://schema.org/VirtualLocation) (un lieu virtuel).
+Saisissez les informations identifiant et décrivant l'endroit où est présenté l'événement. L'entité imbriquée sous la propriété _location_ peut être de @type [Place](https://schema.org/Place) (un lieu physique) ou de @type [VirtualLocation](https://schema.org/VirtualLocation) (un lieu virtuel).
+
+### [_sameAs_](https://schema.org/sameAs)
+Saisissez les URIs d'_identifiants passerelles_[^3] permettant d'identifier sans ambiguité les entitées imbriquées dans l'entité de type _Event_. 
```

Comment:
I see this is the sameAs of the Event, not the sameAs of the nested entities.

The event's sameAs is missing from the JSON-LD template and should be added.

Reply:
You may add "sameAs" for events to the template but I would not mention it in the readme file just yet. I would rather wait until we have a clear vision of what anticipated values we would like to see and guidance for retrieving these URIs.  Let's first focus on encouraging performing arts organizations to create URIs for their performance. When this practice starts to become more common, then we will encourage secondary-source websites to retrieve these URIs and to populate them sameAs property-value pairs. Until then, I would be happy if secondary-source websites bothered using the url field to provide a pointer to the event organizer's site.

Reply:
J'ai déplacé cette section à la suite de "offer" et j'ai indiqué entre parenthèse dans le heading que cette section se rapporte aux entités imbriquées.